### PR TITLE
Add evpn_vni text to readme

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
* Doc-only changes. There was no documentation for evpn_vni prior to this change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1597,6 +1597,53 @@ Profile name of the Encapsulation. Valid values is string.
...
 --
+### Type: cisco_evpn_vni
+
+Manages Cisco Ethernet Virtual Private Network (EVPN) VXLAN Network Identifier (VNI) configurations of a Cisco device.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I3(1)        | 1.3.0                  |
+| N30xx    | unsupported        | 1.3.0                  |
+| N31xx    | unsupported        | 1.3.0                  |
```
Comment (robert-w-gries) [inline README.md]: `Module Minimum Version` should also be listed as `unsupported` for the 3k's

Comment (chrisvanheuveln) [inline README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -1597,6 +1597,53 @@ Profile name of the Encapsulation. Valid values is string.
...
 
 --
+### Type: cisco_evpn_vni
+
+Manages Cisco Ethernet Virtual Private Network (EVPN) VXLAN Network Identifier (VNI) configurations of a Cisco device.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I3(1)        | 1.3.0                  |
+| N30xx    | unsupported        | unsupported            |
```
Comment (mikewiebe) [inline README.md]: Remind me why this is unsupported on N3K?

Comment (chrisvanheuveln) [inline README.md]: h/w does not support nv overlay

**Comment by mikewiebe** [comment]
:+1:

**Comment by robert-w-gries** [comment]
👍

**Pull request merged**